tox.ini

```diff
@@ -8,13 +8,14 @@ passenv =
     CI
 
 deps =
-    django111: Django>=1.11,<2.0
-    -rtest_requirements.txt
+    django22: Django>=2.2,<3.0
+    django30: Django>=3.0,<3.1
+    -r test_requirements.txt
 
 commands =
-    py.test {posargs}
+    pytest {posargs}
     coverage xml
     codecov
-    pylint edx_sga
+    pylint --py3k edx_sga
```

Comment:
This runs just the Python 3.x compatibility checks, which I don't think we need anymore.  Probably should leave this line as it was.

Reply:
Removed this and fixed the quality failures that came without it.